app/deploy/generator/04-syndesis-oauth-proxy.yml.mustache

```diff
@@ -117,6 +120,10 @@
               memory: 200Mi
             requests:
               memory: 20Mi
+{{#Restricted}}
+        serviceAccount: syndesis-oauth-client
+        serviceAccountName: syndesis-oauth-client
```

Comment:
A small nitpicking: `serviceAccount` has been deprecated in favour of `serviceAccountName`, see [the spec](https://docs.openshift.com/container-platform/3.6/rest_api/kubernetes_v1.html#v1-podspec). Imho we don't need both.